### ISSUE TITLE
[Issue] Model Prefab deletes any user added Actors and Scripts after Reimport in Prefab editor and in Prefab Instance

### ISSUE THREAD

**Issue by ElevenArt** [FlaxEngine/FlaxEngine#3122]
When importing a model as Model Prefab and adding any Actors (Collider, Audio Source, Particle Effect,...) to the Model Prefab hierarchy, it deletes these newly added Actors after reimport. It's a critical bug. 
 
https://github.com/user-attachments/assets/9898d722-9768-41cf-aeb1-a68cf27abe5c

Steps to reproduce:
1. Import model as Model Prefab
2. Add any Actor you want to the Model Prefab hierarchy (Collider, Particle Effect, Audio Source, etc.)
3. Save it
4. Click Reimport and these newly added Actors are deleted

Minimal reproduction project:
[test-model.zip](https://github.com/user-attachments/files/18275247/test-model.zip)

Flax version:
Flax 1.8.2, 1.9, 1.10 December master

**Comment by ElevenArt** [comment]
@mafiesto4 , @Tryibion 

Anything new about this? This is really a critical bug, because there is no easy way to fix this.

In the current state of Model Prefab Reimport, there are only 3 options remain:
1) Add the deleted Actors again with all the previous settings  on all Colliders, Particles Effects, Sound Sources, etc.

2) Insert the ModelPrefab into another Prefab and have the added Actors outside the Model Prefab, but in this case, the hierarchy of sub-actors in the Model Prefab does not make any sense, since nothing can be added to the Model Prefab Hierarchy there, without deleting it after Reimport (I can use it straight away Single Mesh model without hierarchy)

3) Import the Model again as a Model with Split option checked and overwrite it in the Model Prefab Sub-Asset folder, but that is also not the best solution, because the sub-assets models must be deleted and replaced with new ones (which were imported as a Model with split option), but then you have to manually add these sub-asset Models to the ModelPrefab Sub-Actors.

Each of these options is nonsense, because it is unnecessary extra work and all of these 3 options it's complicated.

Is there any way to fix this?

**Comment by mafiesto4** [comment]
As a workaround, I would create another prefab out of model prefab and edit it there.

Meanwhile, we will need to add preservation of such changes (like adding custom actors/scripts).

**Comment by ElevenArt** [comment]
@mafiesto4 Yes, deleting Scripts from sub-actors is another Issue of Model Prefab reimport. I was going to make an issue on github for this, but you know about it, so if these 2 things are resolved, then reimport in Model Prefab should work fine. Thank you for adding this Issue to Flax 1.10 milestone :)

**Comment by mafiesto4** [comment]
Fixed in 05e3149ef4fbac05b71fdf6ae3204936a715fc3b.
